.github/workflows/testing.yml

```diff
@@ -11,9 +11,16 @@ jobs:
       image: rustmath/mkl-rust:1.43.0
       options: --security-opt seccomp=unconfined
     strategy:
+      fail-fast: false
       matrix:
         toolchain:
+          - 1.49.0
           - stable
+        experimental: [false]
```

Comment:
running the testing pipeline is the most involved one and perhaps we exclude nightly from this one as well. Regressions caused by testing of certain nightly versions are 1. hard to debug 2. not saying much about the actual test. Compilation errors, on the other hand, are much easier to understand and fix.